Tools/CISupport/ews-build/steps.py

```diff
@@ -310,12 +310,16 @@ def start(self):
     def add_patch_id_url(self):
         patch_id = self.getProperty('patch_id', '')
         if patch_id:
+            self.setProperty('change_id', patch_id, 'ConfigureBuild')
             self.addURL('Patch {}'.format(patch_id), Bugzilla.patch_url(patch_id))
 
     def add_pr_details(self):
         pr_number = self.getProperty('github.number')
         if not pr_number:
             return
+
+        self.setProperty('change_id', pr_number, 'ConfigureBuild')
```

Comment:
We shouldn't use pr_number for change_id. There can be multiple iterations of a PR, and we would want to uniquely identify the uploaded build for each one of them. We should use hash instead.

Reply:
Yes, sorry, that's an overside, previous versions of this used `github.head.sha`